pkg/engine/source/filesystem.go

```diff
@@ -449,13 +395,17 @@ func ReadQuery(queryDir string) (model.QueryMetadata, error) {
 		aggregation = int(agg.(float64))
 	}
 
+	readExperimental, _ := metadata["experimental"].(string)
```

Comment:
Since you are using an auxiliary function to get the experimental value, maybe you could add  the type casting there also.